src/components/Navbar.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink, Outlet } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import userIcon from '../images/usericon.png';
+import styles from '../Style/Navbar.module.css';
+import navbarstyles from '../Style/Books.module.css';
 
-function Navbar() {
-  return (
-    <nav>
-      <ul>
-        <li>
-          <Link to="/books">Books</Link>
-        </li>
-        <li>
-          <Link to="/categories">Categories</Link>
-        </li>
-      </ul>
+const Navbar = ({
+  path,
+}) => (
+  <>
+    <nav className={styles.panelbg}>
+      <div className={styles.BookstoreCMS}>
+        Bookstore CMS
+      </div>
+      <div>
+        <NavLink className={navbarstyles.BOOKS} to="/Books">
+          <div style={{ color: `${path ? 'black' : 'rgb(165, 162, 162)'}` }}>
```

Comment:
[OPTIONAL] Please follow the best practice and don't use inline styling. You can do it with passing variables. You can refactor the code in Navbar.js below.
```
<div className={`text-color ${path ? 'text-color--active' : ''}`}>
```
And in CSS, you can add these classes:
```
.text-color {
  color: rgb(165, 162, 162);
}

.text-color--active {
  color: black;
}
```